### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ealitten**
:shipit:

### CODE REVIEW

**Review by domvernon — APPROVED** [review]
:shipit:

File: README.md [hunk 1]
```diff
@@ -14,39 +14,50 @@ Our team is:
...
+We defined our initial project aim as: 
+```Making predictions using machine learning```
 
-```
-Making predictions using machine learning
-```
+Since we were all new to machine learning and data science, we started the project with a mini learning project: [predicting who survives the sinking of the Titanic](https://www.kaggle.com/c/titanic). This is a popular starting project for ML, and is a good introduction to the simpler ML algorithms. The challenge is a relatively straightforward binary classification problem - there are two outcomes: survived and died. We solved this using a logistic regression algorithm, using three columns from the Titanic dataset (`age`, `sex` and `passenger class`), achieving 80-82% accuracy of prediction. The titanic model and data can be found in the `titanic` folder.
 
-## **MVP**
+Once we'd solved the titanic challenge, we chose a main project: using a dataset of pokemon attributes and battles to predict the winner in a battle between any two Pokemon. The [dataset](https://www.kaggle.com/terminus7/pokemon-challenge) was obtained from kaggle.
```
Comment (Gleoman) [inline README.md]: Very minor but would prefer "we had" to "we'd" in the first sentence.

**Review by Gleoman — APPROVED** [review]
One comment only, otherwise :shipit:

**Review by reenz — APPROVED** [review]
:shipit:

**Pull request merged**